### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,31 +47,7 @@ dotnet add package SwedbankPay.Sdk
...
-## Getting started
-
-Install the `SwedbankPay.Sdk` NuGet in your project:
-
-```
-dotnet add package SwedbankPay.Sdk
-```
-
-Configure the SDK in one line.
+(This requires that you have added `SwedbankPay.Sdk.Extensions`)
```
Comment (asbjornu) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -47,31 +47,7 @@ dotnet add package SwedbankPay.Sdk
 ```
 
 Configure the SDK in one line.
```
Comment (asbjornu) [inline README.md]:
```suggestion
To configure the SDK in one line using `Microsoft.Extensions.DependencyInjection`,
you need to install `SwedbankPay.Sdk.Extensions`. The SDK can then be configured
as such:
```

File: README.md [hunk 3]
```diff
@@ -47,31 +47,7 @@ dotnet add package SwedbankPay.Sdk
...
-
-
-## Getting started
-
-Install the `SwedbankPay.Sdk` NuGet in your project:
-
-```
-dotnet add package SwedbankPay.Sdk
-```
-
```
Comment (asbjornu) [inline README.md]: Please add these lines back, as they are relevant once we've explained that adding `SwedbankPay.Sdk.Extensions` is required.

File: README.md (hunk 1 above)
Comment (NullableInt) [reply README.md]: Without the Extensions NuGet we plan to create there is no way to configure it in one line.

File: README.md (hunk 3 above)
Comment (NullableInt) [reply README.md]: They were duplicated

**Pull request merged**